config/locales/server.en.yml

```
@@ -157,6 +157,10 @@ en:
 
           *Help:* `/discourse help`
         transcript:
+          modal_title: "Build Transcript"
```

```suggestion
          modal_title: "Create Transcript"
```
Build always sounds weird to me - unless this is consistent with some other Slack options?